src/backend/src/routes/work-package-templates.routes.ts

```diff
@@ -28,4 +28,19 @@ workPackageTemplatesRouter.post(
   WorkPackagesController.editWorkPackageTemplate
 );
 
+workPackageTemplatesRouter.post(
+  '/create',
+  nonEmptyString(body('templateName')),
+  nonEmptyString(body('templateNotes')),
+  nonEmptyString(body('workPackageName').optional()),
+  isWorkPackageStageOrNone(body('stage').optional()),
+  intMinZero(body('duration').optional()),
+  body('blockedBy').isArray(),
+  nonEmptyString(body('blockedByInfo.*.name')),
```

Comment:
think this is named something else right?

Reply:
ohhhh this passed validation because blockedbyInfo doesnt exist you need to add a check of nonEmptyString(body(‘blockedBy.*’)